src/libraries/System.IO.Packaging/src/System/IO/Packaging/ZipPackage.cs

```diff
@@ -501,7 +501,7 @@ bool IEqualityComparer<string>.Equals(string? extensionA, string? extensionB)
                 //with the rules for comparing/normalizing partnames.
                 //Refer to PackUriHelper.ValidatedPartUri.GetNormalizedPartUri method.
                 //Currently normalization just involves upper-casing ASCII and hence the simplification.
-                return (string.CompareOrdinal(extensionA.ToUpperInvariant(), extensionB.ToUpperInvariant()) == 0);
+                return extensionA.ToUpperInvariant() == extensionB.ToUpperInvariant();
```

Comment:
Use `extensionA.Equals(extensionB, StringComparison.InvariantCultureIgnoreCase)` instead?

Reply:
Maybe, but as in https://github.com/dotnet/runtime/pull/82438#discussion_r1113349780, I've avoided making any such changes so at to not accidentally break something subtle